lib/bank_account.rb

```diff
@@ -56,26 +56,37 @@ def self.find(id)
     end
 
     def self.associate_everything
-      account_and_owner_csv = CSV.read("support/account_owners.csv")
       everything_array = []
-      account_and_owner_csv.each do |line|
+      CSV.foreach("support/account_owners.csv") do |line|
         each_account = Bank::Account.find(line[0].to_i)
         each_owner = Bank::Owner.find(line[1].to_i)
-        each_account.add_owner(each_owner)
+        each_account.owner = each_owner
         everything_array.push(each_account)
       end
       return everything_array
     end
+
+    def self.find_owner(id)
+      account_and_owner_csv = CSV.read("support/account_owners.csv")
+      owner_csv_array = CSV.read("support/owners.csv")
+      found_line = account_and_owner_csv.find do |line|
+         line[0].to_i == id
+      end
+      found_owner = owner_csv_array.find do |line|
+        line[0].to_i == found_line[1].to_i
+      end
+      return found_owner
+    end
   end
 
   class Owner
     attr_reader :account, :owner_id, :last_name, :first_name, :address, :city, :state
     def initialize(id, last_name, first_name, address, city, state)
-      @account = account
-      @first_name = first_name
-      @last_name = last_name
-      @address =  address
-      @owner_id = owner_id
+      #@account = account
```

Comment:
Hmmm what happened with the owner that made you decide to comment the attributes out?